features/manage_meeting_rooms.feature

```diff
@@ -1,8 +1,13 @@
 Feature: Manage Meeting Rooms
-  In order to create meeting rooms 
+  In order to create meeting rooms
   As an Admin
   I want to create and delete meeting rooms
 
+  Scenario: Destroy Meeting Room
+    When I send a DELETE request to "/v1/meeting_rooms/" with id
+    Then the response status should be "204"
+    And the same meeting_room can not find
```

Comment:
Specify the name of the meeting room or the id. This step is too vague and would be difficult to expand.